runtime/src/main/java/org/corfudb/runtime/Backup.java

```diff
@@ -54,6 +59,17 @@
     // All tables in Corfu Db
     private List<UUID> allTablesInDb;
 
+    // Metrics
+    private final Optional<DistributionSummary> backupTimeDistributionSummary =
+            MeterRegistryProvider.getInstance().map(registry ->
+                    DistributionSummary.builder("backup.table.time")
+                            .baseUnit("ms").register(registry));
+
+    private final Optional<DistributionSummary> backupEntryCountDistributionSummary =
+            MeterRegistryProvider.getInstance().map(registry ->
+                    DistributionSummary.builder("backup.table.entryCount")
```

Comment:
Add 
publishPercentiles(0.50, 0.99)
.publishPercentileHistogram()

Distributions usually have very long tails, which are better captured with percentiles. Otherwise, you would just see mean and max, and you will not be able to accurately judge what's going on.

Reply:
Added. Thanks.